config/config.py.template

```diff
@@ -443,18 +443,52 @@ MLLIB_TESTS += [("glm-classification", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
     MLLIB_JAVA_OPTS, [ConstantOption("glm-classification")] +
     MLLIB_GLM_CLASSIFICATION_TEST_OPTS)]
 
-NAIVE_BAYES_TEST_OPTS = MLLIB_REGRESSION_CLASSIFICATION_TEST_OPTS + [
-    # Expected fraction of examples which are negative
-    OptionSet("per-negative", [0.3]),
-    # The scale factor for the noise in feature values
-    OptionSet("scale-factor", [1.0]),
-    # Naive Bayes smoothing lambda.
-    OptionSet("nb-lambda", [1.0])
-]
 
-MLLIB_TESTS += [("naive-bayes", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
-    MLLIB_JAVA_OPTS, [ConstantOption("naive-bayes")] +
-    NAIVE_BAYES_TEST_OPTS)]
+
+if MLLIB_SPARK_VERSION >= 1.4:
+    NAIVE_BAYES_TEST_OPTS_MULTINOMIAL = MLLIB_REGRESSION_CLASSIFICATION_TEST_OPTS +  [
+        # Expected fraction of examples which are negative
+        OptionSet("per-negative", [0.3]),
+        # The scale factor for the noise in feature values
+        OptionSet("scale-factor", [1.0]),
+        # Naive Bayes smoothing lambda.
+        OptionSet("nb-lambda", [1.0]),
+        # MLLIB_REGRESSION_CLASSIFICATION_TEST_OPTS + [
+        OptionSet("model-type", ["Multinomial"]),
+    ]
+
+    NAIVE_BAYES_TEST_OPTS_BERNOULLI = MLLIB_REGRESSION_CLASSIFICATION_TEST_OPTS + [
+        # Expected fraction of examples which are negative
+        OptionSet("per-negative", [0.3]),
+        # The scale factor for the noise in feature values
+        OptionSet("scale-factor", [1.0]),
+        # Naive Bayes smoothing lambda.
+        OptionSet("nb-lambda", [1.0]),
+        # MLLIB_REGRESSION_CLASSIFICATION_TEST_OPTS + [
+        OptionSet("model-type", ["Bernoulli"]),
+    ]
+
+    MLLIB_TESTS += [("naive-bayes-multinomial", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
```

Comment:
Multinomial naive Bayes test should use the same name as before to compare performance. So we only need

~~~
if MLLIB_SPARK_VERSION >= 1.4:
    NAIVE_BAYES_TEST_OPTS_BERNOULLI ...
    MLLIB_TESTS += ...
~~~